src/main/resources/logback.xml

```diff
@@ -0,0 +1,11 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="TRACE">
+        <appender-ref ref="STDOUT" />
+    </root>
+</configuration>
```

Comment:
Your code is correct, as Logback can handle XML configuration files that don't have an explicit EOF marker. However, it may be a good practice to add an EOF marker with a newline character at the end of the file to ensure compatibility with all text editors and IDEs. Can you please add this as a best practice?